dasharo-compatibility/dasharo-tools-suite.robot

```diff
@@ -0,0 +1,150 @@
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS002.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS002.001 not supported
+    # Not supported due to SSH authorization error while sending the report
+    Skip
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Check HCL Report Creation
+
+DTS003.001 DTS option power-off DUT works correctly
+    [Documentation]    This test aims to verify that the option Power off
+    ...    system in the DTS menu turns off the DUT.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS003.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS003.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Sleep    3s
+    Write Into Terminal    10
+    Check Power Off In DTS
+
+DTS004.001 DTS option reboot DUT works correctly
+    [Documentation]    This test aims to verify that the option Reboot system
+    ...    in the DTS menu reboots the DUT.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS004.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS004.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Sleep    3s
+    Write Into Terminal    11
+    Read From Terminal Until    ${TIANOCORE_STRING}
+
+DTS005.001 DTS drop-to-shell option works correctly
+    [Documentation]    This test aims to verify that Shell can be accessed via
+    ...    enabling SSH in the menu.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS005.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS005.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    # Enable SSH in DTS menu
+    Write Into Terminal    8
+    Login To Linux Via SSH Without Password    root    root@DasharoToolsSuite
+
+DTS006.001 Flash device from DTS shell by using flashrom works correctly
+    [Documentation]    This test aims to verify whether is the possibility to
+    ...    flash the DUT firmware by using flashrom in DTS Shell.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS006.001 not supported
+    Skip If    not ${DTS_FIRMWARE_FLASHING_SUPPORT}    DTS006.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Enter Shell In DTS
+    Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
+    Set DUT Response Timeout    320s
+    Flash Firmware In DTS
+    Power On
+    Enable Network Boot
+    Boot Dasharo Tools Suite    iPXE
+    Enter Shell In DTS
+    Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
+    Check Firmware Version
+
+DTS007.001 Update device firmware from DTS Shell by using fwupd works correctly
+    [Documentation]    This test aims to verify whether there is the
+    ...    possibility to update the DUT firmware by using fwupd in DTS.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS007.001 not supported
+    Skip If    not ${DTS_FWUPD_FIRMWARE_UPDATE_SUPPORT}    DTS007.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Enter Shell In DTS
+    Fwupd Update
```

Comment:
I guess you have not tested that since we have zero devices supporting this test case. I recommend dropping it, as well as the fwupd related keywords.

Reply:
Case [removed](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/c25cb497706d8a74c04839a38cf75f2ad65e1a61#diff-4d0e5cbb7f43565e1c6904f6f0c660951a6f60f7d187a787aa23d8fcab380777L103)